Doppler.HtmlEditorApi.Test/DopplerHtmlDocumentTest.cs

```diff
@@ -0,0 +1,49 @@
+using Doppler.HtmlEditorApi.Storage.DapperProvider.Queries;
+using Doppler.HtmlEditorApi.Storage.DapperProvider;
+using System.Net.Http.Headers;
+using System.Net.Http.Json;
+using System.Net;
+using System.Threading.Tasks;
+using Doppler.HtmlEditorApi.Test.Utils;
+using Microsoft.AspNetCore.Mvc.Testing;
+using Microsoft.Extensions.DependencyInjection;
+using Moq;
+using System.Text.Json;
+using Xunit.Abstractions;
+using Xunit;
+using System.Linq;
+
+namespace Doppler.HtmlEditorApi;
+
+public class DopplerHtmlDocumentTest
+{
+    private readonly ITestOutputHelper _output;
+
+    public DopplerHtmlDocumentTest(ITestOutputHelper output)
+    {
+        _output = output;
+    }
+
+    [Fact]
+    public void get_distinct_fields_id()
```

Comment:
What do you think about this name?

```suggestion
    public void GetFieldsId_should_return_distinct_field_ids()
```